### ISSUE TITLE
Rule: Indentation of param list after a function call

### ISSUE THREAD

**Issue by kevinSuttle** [eslint/eslint#4659]
Style rule. No errors. Most likely custom, but wanted to check first.

1. v1.10.3
2. 
```javascript
const foo = bar(
  param1,
  param2,
  param3
);
```
3. No output
4. Enforcing rule of indented param lists

Is this a thing?

**Comment by eslintbot** [comment]
@kevinSuttle Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
Duplicate https://github.com/eslint/eslint/issues/1801

**Comment by kevinSuttle** [comment]
Thanks @gyandeeps!
